back/src/main/java/com/coffefreaks/timberwallee/controller/MoveController.java

```diff
@@ -0,0 +1,43 @@
+package com.coffefreaks.timberwallee.controller;
+
+import com.coffefreaks.timberwallee.exception.TimberResourceNotFoundException;
+import com.coffefreaks.timberwallee.model.Request.LocationRequest;
+import com.coffefreaks.timberwallee.model.Response.LocationResponse;
+import com.coffefreaks.timberwallee.service.Interface.RobiotService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(value = "/move")
+public class MoveController {
+
+    private final RobiotService robiotService;
+
+    /***
+     * Constructor
+     * @param robiotService a RobiotService
+     */
+    @Autowired
+    public MoveController(RobiotService robiotService) {
+        this.robiotService = robiotService;
+    }
+
+    /**
+     * Unit move for robiot
+     * @param locationRequest a locationRequest object
+     * @return moveResponse
+     */
+    @PostMapping
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public LocationResponse unitMove(@Valid @RequestBody LocationRequest locationRequest) {
+
+        boolean results = this.robiotService.move(locationRequest);
+        if (!results) {
```

Comment:
Pourquoi tu fais un not sur le résultat du service ?
RobiotService.move renvoie true si le déplacement est ok